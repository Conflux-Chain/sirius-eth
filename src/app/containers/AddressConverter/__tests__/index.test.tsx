import React from 'react';
import { render } from '@testing-library/react';
import { HelmetProvider } from 'react-helmet-async';
import { MemoryRouter } from 'react-router';

import { AddressConverter } from '..';

const renderComponent = () =>
  render(
    <HelmetProvider>
      <MemoryRouter>
        <AddressConverter />
      </MemoryRouter>
    </HelmetProvider>,
  );

describe('<AddressConverter />', () => {
  it('should match the snapshot', () => {
    const component = renderComponent();
    expect(component.container.firstChild).toMatchSnapshot();
  });
});
