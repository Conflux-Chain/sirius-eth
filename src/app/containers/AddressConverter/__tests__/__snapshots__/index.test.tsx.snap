// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddressConverter /> should match the snapshot 1`] = `
.c0 {
  font-size: 1.7143rem;
  font-weight: 500;
  color: #1a1a1a;
  line-height: 2.2857rem;
  margin-bottom: 1.1429rem;
  margin-top: 2.2857rem;
}

.c1 {
  font-size: 1rem;
  color: #74798c;
  margin-top: 0.8571rem;
  line-height: 1.2857rem;
}

@media (max-width:600px) {
  .c0 {
    font-size: 1.2857rem;
    font-weight: 500;
    color: #050505;
    line-height: 1.8571rem;
  }
}

@media (max-width:600px) {

}

@media (max-width:600px) {

}

@media (max-width:600px) {

}

<div
  class="c0"
>
  Address Format Conversion
  <div
    class="c1"
  />
</div>
`;
